### PR TITLE
Add helper PCBs: v7 tested + 6" adapters

### PR DESCRIPTION
**Pull request by martinberlin**
This PR adds two very simple boards in the folder **helper-PCBs**:

1. extension-9_40_to_34_6inch: It's an adapter from 33 pin (9.7") and 40 pins in two panelized small boards, both with a 6 inch output FPC (34 pins)
2. extension-40_to_PCB_test: It's a very simple idea, to route all signals, plus 16 bit output to Resistance + LED. This will allow testing your soldering very easily at least to know all driving signals and data GPIOs are well soldered everywhere in your PCB.

### CODE REVIEW

**Comment by vroland** [comment]
Hey, nice, both sound pretty useful :) Just some notes to make it compatible with the automation:

* For every board, the KiCAD project should be given a unique name, and have the same name as the containing folder. E.g., `epdiy-v7/epdiy-v7.{kicad_pro,kicad_pcb,kicad_sch}`. This can easily be renamed by selecting `File -> Save As` in the main KiCAD window and saving it to a folder with the new name.
* I'd make separate sub-folders for `helpers` and `adapters`. Just to organize it a bit by purpose :) 
* Delete the manufacturing files and keep only the sources. Since we generate those automatically, having potentially outdated ones in the repo will be confusing. 
* Each board needs a README.md in it's project folder. Just refer to `epdiy-v7/README.md` for an example.

I should also put this into a contribution guide / PR template, but haven't gotten to that, yet ;)

**Comment by martinberlin** [comment]
Great to know more about the compatibility with the automation. I think didn't forgot anything but if I did just let me know.
This is how it looks the PCB tester:
PCB tester
At the moment of designing it I was also testing MicroSD. I though sometimes when you need to do small cheap tests is great to panelize two or 3 maximum like this so you just reduce costs. Then if you need then it's easy just to cut both in the middle and separate them. 
But at least is only one PCB price and sent in the same package.

**Comment by vroland** [comment]
Thanks for updating! There are still manufacturing files for the second board, can you delete them?
Otherwise this looks good to me :) Since it's only a helper PCB, just leaving the SDCard test is fine, if you don't need it not populating it is basically free.

File: helpers/40_pins_PCB_tester/PCB-test-Schematic.pdf
Comment (vroland) [inline helpers/40_pins_PCB_tester/PCB-test-Schematic.pdf:1]: delete this, will be auto generated

File: adapters/9.7_40_to_34_6inch/adapter_schematic.pdf
Comment (vroland) [inline adapters/9.7_40_to_34_6inch/adapter_schematic.pdf:1]: delete this, will be auto generated

**Review by vroland — APPROVED** [review]
Good call on adding the silk screen mention!

File: helpers/40_pins_PCB_tester/PCB-test-Schematic.pdf
Comment (martinberlin) [reply helpers/40_pins_PCB_tester/PCB-test-Schematic.pdf:1]: git rm helpers/40_pins_PCB_tester/PCB-test-Schematic.pdf   Done!

File: adapters/9.7_40_to_34_6inch/adapter_schematic.pdf
Comment (martinberlin) [reply adapters/9.7_40_to_34_6inch/adapter_schematic.pdf:1]: git rm adapters/9.7_40_to_34_6inch/adapter_schematic.pdf Done!

**Pull request merged**